engine.py

```diff
@@ -198,6 +209,113 @@ def log_warning(self, msg):
         print "Shotgun Warning: %s" % msg
 
     ############################################################################
+    # panel interfaces
+    ############################################################################
+
+    def get_panel_info(self, requested_panel_id):
+        """Get info dict for the panel with the supplied id."""
```

Comment:
i don't necessarily feel that the `requested_panel_id` param needs to be documented, it seems self explanatory, but I think it would be useful to document the return value here.